app/styles/modules/_global_header.scss

```diff
@@ -1,39 +1,57 @@
-header {
-  background: #4d4d4d;
+#global-header {
+  @include color-scheme($white, $middle-grey);
   position: fixed;
   top: 0;
   z-index: 100;
 
   .row-inner {
-    height: 65px;
+    height: $double-grid;
   }
 
   h1 {
-    @include vertical-align;
-    color: #ffffff;
-    font-size: 24px;
-    margin: 0;
+    flex: 1;
+    font-size: $large-font-size;
+    line-height: $large-font-size;
+    margin: 0 $default-gap;
   }
 
-  #user {
-    display: none;
+  #user { 
+    background: $baby-grey;
+    border-radius: 50%;
+    font-size: 0px;
+    height: $default-grid;
+    min-width: $default-grid;
   }
 
   #search {
-    margin-top: -20px;
-    position: absolute;
-    right: 20px;
-    top: 50%;
-
-    input {
-      @include hidpi-background-image(icon-search, 16px 16px);
-      background-position: top 50% right 10px;
-      background-repeat: no-repeat;
-      border: 0;
-      border-radius: 6px;
-      font-size: 14px;
-      padding: 7px 7px 7px 15px;
-      width: 350px;
+    flex-basis: $default-grid * 10;
+      input {
```

Comment:
Indenting looks "off" here. Me thinks that `input {` should be the same indentation level as `flex-basis`. But whatevah.